nilearn/plotting/html_surface.py

```diff
@@ -68,7 +68,7 @@ def _remove_after_n_seconds(file_name, n_seconds):
     subprocess.Popen(['python', script, file_name, str(n_seconds)])
 
 
-class SurfaceView(object):
+class HTMLDocument(object):
```

Comment:
The fact that "_check_html" and "HTMLDocument" are in the file "html_surface" while they have nothing to do with surfaces is confusing. I believe that either all files related to html plotting should be merged in one, or they should be split in 3.